SourceX/controls/game_controls.h

```diff
@@ -22,7 +25,7 @@ enum GameActionType {
 };
 
 struct GameActionSendKey {
-	DWORD vk_code;
+	std::uint32_t vk_code;

```

Comment:
Would prefer Uint32

Reply:
I think we should use standard C++ types instead of the SDL stuff that was added for compatibility with ancient C versions (not C++)?

Reply:
Uint32 maps to uint32_t on systems where it is available, so would there be much of a difference in the cases you refer to?

Reply:
We don't support any compilers where it's not available and we never will
`<cstdint>` is ancient stuff

Reply:
SDL just happens to be even more ancient

Reply:
Anyway, if we change something like that it should be done in a separate PR throughout.

Changed to Uint32